src/pages/index.tsx

```diff
@@ -19,7 +19,7 @@ export const itemsPerPage = 6;
 const dataClient = new DataClient(new SanityClient(), new OpenDataClient());
 
 export async function getStaticProps() {
-  const data = await dataClient.getAllSiteNotices(itemsPerPage, 0);
+  const data = await dataClient.getAllSiteNotices(0, undefined, itemsPerPage);
```

Comment:
if location was the last param for getAllSiteNotices could we avoid the need to pass undefined?